sbi/inference/__init__.py

```diff
@@ -1,42 +1,45 @@
 from sbi.inference.abc import MCABC, SMCABC
-from sbi.inference.base import (
+from sbi.inference.trainers.base import (
     NeuralInference,  # noqa: F401
     check_if_proposal_has_default_x,
     infer,
     simulate_for_sbi,
 )
-from sbi.inference.fmpe import FMPE
-from sbi.inference.npse.npse import NPSE
-from sbi.inference.snle import MNLE, SNLE_A
-from sbi.inference.snpe import SNPE_A, SNPE_B, SNPE_C  # noqa: F401
-from sbi.inference.snre import BNRE, SNRE, SNRE_A, SNRE_B, SNRE_C  # noqa: F401
+from sbi.inference.trainers.fmpe import FMPE
+from sbi.inference.trainers.nle import MNLE, NLE_A
+from sbi.inference.trainers.npe import NPE_A, NPE_B, NPE_C  # noqa: F401
+from sbi.inference.trainers.npse.npse import NPSE
```

Comment:
why do we need the extra `npse` here?

Reply:
and for `fmpe` we don't?

Reply:
both works, because `sbi.inference.trainers.npse.__init__.py` contains an import for NPSE. But I now made it the same.